readme.md

```diff
@@ -190,6 +192,15 @@ merge(
 ) // => [{ a: true, b: true }, 'ah yup']
 ```
 
+### `arrayMerge: customFunction`
+
+If you want to supply your own custom method then pass it as an option to `arrayMerge`
+
+```js
+const customFunction = ...
```

Comment:
This is why the tests are failing - that's not valid JS syntax.  All the JS code in the readme is tested to make sure the examples are all correct.